src/application/components/Queries/QueryViewer.tsx

```diff
@@ -53,19 +47,30 @@ export const queryStringHeader = css`
 `;
 
 const queryDataHeader = css`
+  display: flex;
   grid-area: queryDataHeader;
   height: 100%;
   background-color: transparent;
   ${headerStyles}
   color: ${colors.grey.lighter};
 
   button {
-    padding-top: 0;
-    padding-bottom: 1.7rem;
+    padding: 0 0.5em 1.7rem;
```

Comment:
Nit: mixing `em` and `rem`. As an aside, I'd be in favor of migrating to Tailwind or a similar utility class-based system at some point in the future :)

Reply:
Ah ya good callout.

My goal with this PR wasn't so much to fix CSS issues (like this), but rather ensure compatibility with the existing CSS. This used to use [`@reach/tabs`](https://reach.tech/tabs) which included its own [styling](https://github.com/reach/reach-ui/blob/dev/packages/tabs/styles.css). The `0.5em` value came from the [`@reach/tabs` stylesheet](https://github.com/reach/reach-ui/blob/43f450db7bcb25a743121fe31355f2294065a049/packages/tabs/styles.css#L21), which I blindly copied over 😄.

I might leave this for now and would love to come back through this with a fine-tooth comb later on. I'm sure there are many other areas like this that could use improvement.